HACKING.rst

```diff
@@ -31,7 +31,7 @@ you may need to run::
 
   pip install importlib
 
-You may get an error 'psycopg2 package not found' when you do 'haas init_db' 
+You may get an error 'psycopg2 package not found' when you do 'haas-admin db create'
```

Comment:
What the motivation behind this change. Does it really create a database or populates an already created database ? (in case of postgresql)